internal/exec/stages/disks/luks.go

```diff
@@ -531,3 +564,151 @@ func (d LuksDump) hasFlag(flag string) bool {
 	}
 	return false
 }
+
+// collect the cex card domains xx.xxxx
+func getAllCexOnlineAPQNs() (string, error) {
+	apDir := "/sys/bus/ap/devices/"
+	var ret string
+	// cards details are in form of link file points to card directory.
+	event, _ := os.ReadDir(apDir)
+	if len(event) == 0 {
+		return "", fmt.Errorf("adjunct processor not found")
+	}
+	// match the regexp for pattern xx.xxxx
+	for _, dir := range event {
+		match := cexRegx.MatchString(dir.Name())
+		// if a match is found, read the uevent from that APQN directory
+		// to verify whether it corresponds to a CCA controller. This is
+		// intended for potential future use with an EP11 controller.
+		if match {
```

Comment:
Minor: could use the same trick here of `if !match { continue }` to avoid an indent.

Also, might as well get rid of the `match` variable and just work with `cexRegx.MatchString(...)` directly.

Reply:
Done